kolibri/plugins/user/assets/src/routes.js

```diff
@@ -52,6 +52,9 @@ export default [
         if (store.getters.facilities.length > 1 && !store.state.facilityId) {
           // Go to FacilitySelect with whereToNext => SignUpPage
           const whereToNext = router.getRoute(ComponentMap.SIGN_IN);
+          if ((to.query || {}).next) {
```

Comment:
```suggestion
          if (to.query.next) {
```

I'm pretty sure `to.query` is always defined. Did you get errors if you leave out the default `{}`?